packages/Babylonian-Core.package/BPMethodExample.class/instance/execute.st

```diff
@@ -1,6 +1,11 @@
 tracing
 execute
 	
-	^ self newReceiver 
-		perform: self method selector 
-		withArguments: self newArguments
\ No newline at end of file
+	|executionResult|
```

Comment:
The code for recording the execution result should actually be in BlockClosure>>#bpTraceWith: tracer and should make use of `BPActiveTracer value` to ensure that the trace is constructed correctly.

Reply:
Which is the code used in bpTraceExecutionResult: :)